geowebcache/rest/src/test/java/org/geowebcache/rest/controller/SeedControllerTest.java

```diff
@@ -89,12 +91,32 @@ public void testSeedLayerNameWithDots() throws Exception {
         String layerName = "test:mock.layer.name";
         // when(formService.handleFormPost(eq(layerName),
         // anyMap())).thenReturn(ResponseEntity.ok(null));
+        testGet(layerName);
+        testPost(layerName);
+    }
+
+    private void testPost(String layerName) throws Exception {
         doReturn(ResponseEntity.ok(null)).when(formService).handleFormPost(anyString(), anyMap());
+
         mockMvc.perform(
                         post("/rest/seed/{layer}", layerName)
                                 .contentType(MediaType.APPLICATION_FORM_URLENCODED)
                                 .accept(MediaType.TEXT_XML))
                 .andExpect(status().isOk());
         verify(formService).handleFormPost(eq(layerName), anyMap());
     }
+
+    private void testGet(String layerName) throws Exception {
+        doReturn(ResponseEntity.ok(null))
+                .when(formService)
+                .handleGet(any(HttpServletRequest.class), anyString());
+
+        mockMvc.perform(
+                        get("/rest/seed/{layer}", layerName)
+                                .contentType(MediaType.TEXT_PLAIN)
+                                .accept(MediaType.TEXT_HTML))
```

Comment:
Since XML is now going through a different method, a seperate XML test for dots in name may be warranted, per the question above

Reply:
Added tests for that

> Since XML is now going through a different method, a seperate XML test for dots in name may be warranted, per the question above

Added tests